ipnsfs/file.go

```diff
@@ -0,0 +1,124 @@
+package ipnsfs
+
+import (
+	"errors"
+	"io"
+	"os"
+
+	chunk "github.com/jbenet/go-ipfs/importer/chunk"
+	dag "github.com/jbenet/go-ipfs/merkledag"
+	mod "github.com/jbenet/go-ipfs/unixfs/mod"
+
+	context "github.com/jbenet/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+)
+
+type File interface {
+	io.ReadWriteCloser
+	io.WriterAt
+	Seek(int64, int) (int64, error)
+	Size() (int64, error)
+	Flush() error
+	Truncate(int64) error
+	FSNode
+}
+
+type file struct {
+	parent childCloser
+	fs     *Filesystem
+
+	name       string
+	hasChanges bool
```

Comment:
Do you have a `write>seek back>read` test? `hasChanges` seems racy at first thought?